consensus/enclave/impl/src/lib.rs

```diff
@@ -938,13 +953,21 @@ impl ConsensusEnclave for SgxConsensusEnclave {
             mint_txs,
         };
 
+        let timestamp = if config.block_version.timestamps_are_supported() {
+            Self::validate_timestamp(inputs.timestamp, parent_block)?
+        } else {
+            // This should be zero in these instances, Block::new_with_parent()
+            // will panic if it's not.
+            inputs.timestamp
+        };
```

Comment:
We may need to rethink this here.
The byzantine_ledger/worker.rs fills out the timestamps and provides them here, but it doesn't know what the block version the enclave is going to be using.
It may be that we should use zero instead of inputs.timestamp here.

Reply:
Hmm, yeah, that seems reasonable to me (unless we plumb the block version into the byzantine ledger worker but it seems too heavy handed to do that just for this.

Reply:
Using 0 now, added a comment in this location as well as in the field description of `FormBlockInputs`